### PR TITLE
翻译 模型选择和 监督学习

### PR DESCRIPTION
**Pull request by Zoctan**
翻译得可能不太好+_+

### CODE REVIEW

File: doc/zh/tutorial/statistical_inference/supervised_learning.rst [hunk 1]
```diff
@@ -1,79 +1,57 @@
...
+K 个最近邻分类器
 -------------------------------
 
-The simplest possible classifier is the
-`nearest neighbor <https://en.wikipedia.org/wiki/K-nearest_neighbor_algorithm>`_:
-given a new observation ``X_test``, find in the training set (i.e. the data
-used to train the estimator) the observation with the closest feature vector.
-(Please see the :ref:`Nearest Neighbors section<neighbors>` of the online
-Scikit-learn documentation for more information about this type of classifier.)
+[`最近邻`](https://en.wikipedia.org/wiki/K-nearest_neighbor_algorithm) 也许是最简单的分类器：给定一个新的观察值 ``X_test``，用最接近的特征向量在训练集（比如，用于训练估计器的数据）找到观察值。（请看 Scikit 在线学习文档的 [`最近邻部分`]()获取更多关于这种分类器的信息）
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/supervised_learning.rst]: 直接将英文变成中文，格式无需改变

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 2]
```diff
@@ -1,291 +1,272 @@
...
+...     y_train = np.concatenate(y_train)
+...     scores.append(svc.fit(X_train, y_train).score(X_test, y_test))
+>>> print(scores)
+[0.93489148580968284, 0.95659432387312182, 0.93989983305509184]
+```
 
 .. currentmodule:: sklearn.model_selection
 
-This is called a :class:`KFold` cross-validation.
+这被称为：类：`KFold` 交叉验证.
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: :class: 这种无需更改，保持原样即可，还有，英文状态下的冒号保持原样，就英文的就好

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 3]
```diff
@@ -1,291 +1,272 @@
...
+>>> k_fold = KFold(n_splits=3)
+>>> for train_indices, test_indices in k_fold.split(X):
+...      print('Train: %s | test: %s' % (train_indices, test_indices))
+Train: [2 3 4 5] | test: [0 1]
+Train: [0 1 4 5] | test: [2 3]
+Train: [0 1 2 3] | test: [4 5]
+```
 
-The cross-validation can then be performed easily::
+然后就可以很容易地执行交叉验证了。
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: 后面的 :: 保留一下好一点 ^_^

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 4]
```diff
@@ -1,291 +1,272 @@
...
+...          for train, test in k_fold.split(X_digits)]
+[0.93489148580968284, 0.95659432387312182, 0.93989983305509184]
+```
 
-The cross-validation score can be directly calculated using the
-:func:`cross_val_score` helper. Given an estimator, the cross-validation object
-and the input dataset, the :func:`cross_val_score` splits the data repeatedly into
-a training and a testing set, trains the estimator using the training set and
-computes the scores based on the testing set for each iteration of cross-validation.
+交叉验证分数可以使用函数： `cross_val_score` 直接计算出来。给定一个估计量，交叉验证对象，和输入数据集，`cross_val_score` 函数就会反复分解出训练和测试集的数据，然后使用训练集和为每次迭代交叉验证运算出的基于测试集的分数来训练估计量。
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: :func: 这种保持原样即可

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 5]
```diff
@@ -1,291 +1,272 @@
...
 
-By default the estimator's ``score`` method is used to compute the individual scores.
+默认情况下，估计器的 ``score`` 方法被用于运算个体分数。
 
-Refer the :ref:`metrics module <metrics>` to learn more on the available scoring
-methods.
+可以参考 `metrics 模块<metrics>` 学习更多可用的评分方法。
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: :ref: 保持原样即可

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 6]
```diff
@@ -1,291 +1,272 @@
...
-   **Cross-validation generators**
+   **交叉验证生成器**
 
 
 .. list-table::
 
    *
 
-    - :class:`KFold` **(n_splits, shuffle, random_state)**
+    - 类：`KFold` **(n_splits, shuffle, random_state)**
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: :class:

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 7]
```diff
@@ -1,291 +1,272 @@
...
 
-    - :class:`StratifiedKFold` **(n_splits, shuffle, random_state)**
+    - 类：`StratifiedKFold` **(n_splits, shuffle, random_state)**
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: :class:

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 8]
```diff
@@ -1,291 +1,272 @@
...
 
-    - :class:`GroupKFold` **(n_splits)**
+    - 类：`GroupKFold` **(n_splits)**
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: :class:

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 9]
```diff
@@ -1,291 +1,272 @@
...
-   On the digits dataset, plot the cross-validation score of a :class:`SVC`
-   estimator with an linear kernel as a function of parameter ``C`` (use a
-   logarithmic grid of points, from 1 to 10).
+   在数字数据集中，用一个线性内核绘制一个类：`SVC` 估计器的交叉验证分数来作为 ``C`` 参数函数（使用从1到10的点对数网格）.
 
    .. literalinclude:: ../../auto_examples/exercises/plot_cv_digits.py
        :lines: 13-23
 
-   **Solution:** :ref:`sphx_glr_auto_examples_exercises_plot_cv_digits.py`
+   **方法：** [`sphx_glr_auto_examples_exercises_plot_cv_digits.py`](http://scikit-learn.org/stable/auto_examples/exercises/plot_cv_digits.html#sphx-glr-auto-examples-exercises-plot-cv-digits-py)
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: 这个地方保留原样就好啦~~~

File: doc/zh/tutorial/statistical_inference/model_selection.rst [hunk 10]
```diff
@@ -1,291 +1,272 @@
...
+   在糖尿病数据集中，找到最优正则化参数 α。
 
-   **Bonus**: How much can you trust the selection of alpha?
+   **另外：** 你有多相信 α 的选择？
 
    .. literalinclude:: ../../auto_examples/exercises/plot_cv_diabetes.py
        :lines: 17-24
 
-   **Solution:** :ref:`sphx_glr_auto_examples_exercises_plot_cv_diabetes.py`
+   **方法：** [`sphx_glr_auto_examples_exercises_plot_cv_diabetes.py`](http://scikit-learn.org/stable/auto_examples/exercises/plot_cv_diabetes.html#sphx-glr-auto-examples-exercises-plot-cv-diabetes-py)
```
Comment (chenyyx) [inline doc/zh/tutorial/statistical_inference/model_selection.rst]: 这个地方也保留一下原样吧~
